src/content/waypoint/install-landing.json

```
@@ -1,23 +1,11 @@
 {
-	"featuredLearnContent": [
-		{
-			"tutorialSlug": "waypoint/get-started-intro"
-		},
-		{
-			"collectionSlug": "waypoint/get-started-kubernetes"
-		},
-		{
-			"collectionSlug": "waypoint/get-started-nomad"
-		},
-		{
-			"collectionSlug": "waypoint/deploy-aws"
-		},
-		{
-			"collectionSlug": "waypoint/deploy-azure"
-		},
-		{
-			"collectionSlug": "waypoint/deploy-google-cloud"
-		}
+	"featuredTutorialsSlugs": [
+		"waypoint/get-started-intro",
+		"waypoint/get-started-install",
```

This actually refers to the kubernetes guide at `waypoint/tutorials/get-started-kubernetes/get-started-install` but it's picking up the docker one (which is the first linked guide in the list).

Will something like this work?
```suggestion
		"waypoint/get-started-kubernetes/get-started-install",
```

---

@kendallstrautman @BrandonRomano is it possible to specify which collection we'd like to link to here if it's not a tutorial's default collection? These tutorial slugs are passed to `featuredTutorialsSlugs`, which calls `getTutorials`. I'm not familiar enough with that endpoint to know if it will accept a tutorial slug that includes information about the collection.